app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkChaptersCompletedFragmentTest.kt

```diff
@@ -868,6 +921,16 @@ class MarkChaptersCompletedFragmentTest {
     ).check(matches(isChecked()))
   }
 
+  private fun verifyItemUnCheckedOnRecyclerViewItemAtPosition(itemPosition: Int) {
```

Comment:
```suggestion
  private fun verifyItemUncheckedOnRecyclerViewItemAtPosition(itemPosition: Int) {
```

('unchecked' is one word).

Reply:
This appears to have been addressed for chapters & stories, but seems to still be unresolved for MarkTopicsCompletedFragmentTest. Please fix.

Reply:
Ohh don't know how i missed this, have applied a patch for this as well.